csv2table/csv2table.go

```diff
@@ -18,7 +18,6 @@ var (
 	align     = flag.String("a", "none", "Set alignment with eg. none|left|right|center")
 	pipe      = flag.Bool("p", false, "Support for Piping from STDIN")
 	border    = flag.Bool("b", true, "Enable / disable table border")
-	casing    = flag.Bool("c", false, "Set header to be title case")
```

Comment:
Why are you removing support for casting if i may ask

Reply:
Because I believe it's broken, `SetTitleCase()` does not exist.